### ISSUE TITLE
Gradle sync failing when using xcode 16

### ISSUE THREAD

**Issue by plusmobileapps** [getsentry/sentry-kotlin-multiplatform#277]
### Platform

Apple, Android, JVM

### Installed

CocoaPods

### Version

0.9.0

### Steps to Reproduce

Update your xcode tools on your mac to 16.0 and resync your gradle project in Android Studio. 

### Expected Result

IDE should successfully sync 

### Actual Result

IDE failed to sync. Switching back to 15.4 fixed the issue.

**Comment by buenaflor** [comment]
Could you try Kotlin `2.0.21-RC` 

Kotlin/Native is a bit behind in supporting Xcode 16, this is not a Sentry specific issue

**Comment by plusmobileapps** [comment]
That would make sense, is that something that would need to be updated in the docs? Or waiting for the new version of kotlin to stabilize before updating the docs?

**Comment by buenaflor** [comment]
I'll add this to the docs as well

Btw does it work with 2.0.21-RC?

**Comment by plusmobileapps** [comment]
I managed to get this working by updating my kotlin version to `2.0.21` and the sentry cocoapods sdk to `8.38.0`. 

Also worth noting since I updated my computer I had to install the latest xcode which was 16.1, but since kotlin only supports xcode 16 I had to [download](https://developer.apple.com/services-account/download?path=/Developer_Tools/Xcode_16/Xcode_16.xip) that version from apple. After unzipping the older version of xcode, renamed the app to `Xcode_16` and moved it into the applications folder. Finally set this version as your systems xcode in the command line. 

```
sudo xcode-select -s /Applications/Xcode_16.app
```
